src/components/MarkdownProvider/components/Configuration.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { UnorderedList, Span } from '@zendeskgarden/react-typography';
+import { Anchor } from '@zendeskgarden/react-buttons';
+import { getColor } from '@zendeskgarden/react-theming';
+import { ComponentDoc } from 'react-docgen-typescript';
+
+export interface IPackage {
+  version: string;
+  name: string;
+  peerDependencies: string[];
+  packageName: string;
+}
+
+const StyledUnorderedList = styled(UnorderedList)`
+  margin: 0 0 ${p => p.theme.space.xxl};
+  border-bottom: ${p => `${p.theme.borders.sm} ${getColor('grey', 300, p.theme)}`};
+`;
+
+const StyledListItem = styled(UnorderedList.Item)`
+  list-style: none;
+
+  & > div {
+    display: flex;
+    margin: ${p => p.theme.space.sm} 0;
+    border-top: ${p => `${p.theme.borders.sm} ${getColor('grey', 300, p.theme)}`};
+    padding: ${p => p.theme.space.sm} 0 0;
+    font-family: ${p => p.theme.fonts.mono};
+  }
+`;
+
+const StyledListItemLabel = styled.label`
+  min-width: ${p => p.theme.space.base * 20}px;
+  font-family: ${p => p.theme.fonts.system};
+  font-weight: ${p => p.theme.fontWeights.semibold};
+`;
+
+const StyledSpan = styled(Span)`
+  margin-right: ${p => p.theme.space.sm};
+`;
+
+export const Configuration: React.FC<{ reactPackage: IPackage; propSheets: ComponentDoc[] }> = ({
+  reactPackage,
+  propSheets
+}) => {
+  return (
+    <StyledUnorderedList>
+      <StyledListItem>
+        <StyledListItemLabel>Name</StyledListItemLabel>
+        <StyledSpan>{reactPackage.version}</StyledSpan>
+        <StyledSpan>•</StyledSpan>
+        <StyledSpan>
+          <Anchor
+            href={`https://github.com/zendeskgarden/react-components/tree/master/packages/${reactPackage.packageName}`}
+            target="_blank"
+          >
+            View source
+          </Anchor>
+        </StyledSpan>
+        <StyledSpan>•</StyledSpan>
+        <StyledSpan>
+          <Anchor
+            href={`https://www.npmjs.com/package/@zendeskgarden/react-${reactPackage.packageName}`}
+            target="_blank"
+          >
+            View on NPM
+          </Anchor>
+        </StyledSpan>
```

Comment:
Alternatively, we could have a single `<a>` tag without the wrapping `<span>`. We could remove all the `StyledSpan` elements and only style the two dots with horizontal margin of `12px`. It might look something [like this](https://5ed596ca6afa3c0a11941f33--upbeat-fermi-0d23ad.netlify.app/components/avatar#configuration).

Reply:
Much simpler imo. ty 👍